### PR TITLE
Controller

### PR DESCRIPTION
**Pull request by isahwahni**

### CODE REVIEW

File: src/com/generic/setup/Common.java [hunk 1]
```diff
@@ -546,6 +547,40 @@ public static void captureScreen(String screenShotName) {
...
+
+		ArrayList<String> runners = new ArrayList<String>();
+		
+		dataProviderUtils TDP = dataProviderUtils.getInstance();
+		Object[][] data = TDP.getData(SheetVariables.RunnersRegressionSheet);
+		
+		// data map
+		int name = 0;
+
+		logs.debug( data.length+"---------");
```
Comment (ibatta) [inline src/com/generic/setup/Common.java]: remove

File: src/com/generic/setup/Common.java [hunk 2]
```diff
@@ -546,6 +547,40 @@ public static void captureScreen(String screenShotName) {
...
+		
+		for (int row = 0; row < data.length; row++) {
+			runners.add((String) data[row][name]);
+		}
+		return runners;
+	}//read users
```
Comment (ibatta) [inline src/com/generic/setup/Common.java]: tests

File: src/com/generic/setup/Common.java [hunk 3]
```diff
@@ -546,6 +547,40 @@ public static void captureScreen(String screenShotName) {
...
+		Object[][] data = TDP.getData(SheetVariables.BrowsersListingSheet);
+
+		// data map
+		int name = 0;
+
+		for (int row = 0; row < data.length; row++) {
+				browsers.add((String) data[row][name]);
+		}
+		return browsers;
+	}//read users
```
Comment (ibatta) [inline src/com/generic/setup/Common.java]: browsers

File: src/com/generic/tests/runners/BuildFullRegression.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+						new File(System.getProperty("user.dir") + "\\src\\com\\generic\\test_runners"),
+						regressionName));
+			}
+		} catch (Throwable t) {
+			t.printStackTrace();
+		}
+	}
+
+	@Test
+	public void generateFinalregressionXML() {
```
Comment (ibatta) [inline src/com/generic/tests/runners/BuildFullRegression.java]: Regression

**Comment by isahwahni** [comment]
All Required changes are updated

**Pull request merged**